save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ManageDatabaseController.kt

```diff
@@ -0,0 +1,35 @@
+package org.cqfn.save.backend.controllers
+
+import org.cqfn.save.backend.services.ManageDatabaseService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * Controller that is responsible for managing database
+ */
+@RestController("/database")
```

Comment:
I think we already have different controllers hierarchy, and most of them interact with DB